Project.toml

```
@@ -1,24 +1,25 @@
 name = "MLJText"
 uuid = "5e27fcf9-6bac-46ba-8580-b5712f3d6387"
-authors = ["Chris Alexander, Anthony D. Blaom <anthony.blaom@gmail.com>"]
+authors = ["Chris Alexander <uvapazzo@gmail.com>, Anthony D. Blaom <anthony.blaom@gmail.com>"]
 version = "0.1.0"
 
 [deps]
 MLJModelInterface = "e80e1ace-859a-464e-9ed9-23947d8ae3ea"
 ScientificTypesBase = "30f210dd-8aff-4c5f-94ba-8e64358c1161"
+ScientificTypes = "321657f4-b219-11e9-178b-2701a2544e81"
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 TextAnalysis = "a2db99b7-8b79-58f8-94bf-bbc811eef33d"
 
 [compat]
 MLJModelInterface = "1.1.1"
-ScientificTypesBase = "1"
+ScientificTypesBase = "2.2.0"
+ScientificTypes = "2.2.0"
 TextAnalysis = "0.7.3"
```

Wouldn't it better to:
```suggestion
TextAnalysis = "0.7"
```

---

on this want we need the latest for now (0.7.3)